SitePackage.lua

```diff
@@ -234,6 +234,8 @@ local function visible_hook(modT)
         modT.isVisible = false
     elseif modT.fullName:find("EESSI/") then
         modT.isVisible = false
+    elseif modT.fullName:find("JupyterHub/") then
```

Comment:
can you bump the release too?

Reply:
sure, done in https://github.com/vub-hpc/Lmod-UGent/pull/23/commits/06b6f518f02ed41f1a5382ae2d9229cab6e7329c